src/main/res/layout/activity_navigation.xml

```diff
@@ -19,21 +19,47 @@
   -->
 
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
 	android:id="@+id/drawer_content"
 	android:layout_width="match_parent"
 	android:layout_height="match_parent"
 	android:animateLayoutChanges="true">
 
-	<FrameLayout
+	<RelativeLayout
 		android:id="@+id/container_content"
 		android:layout_width="match_parent"
-		android:layout_height="match_parent"/>
+		android:layout_height="match_parent">
+
+		<android.support.v7.widget.Toolbar
+			android:layout_height="?attr/actionBarSize"
+			android:layout_width="match_parent"
+			android:background="@drawable/bg_action_bar"
+			android:id="@+id/toolbar"
+			android:minHeight="?attr/actionBarSize"
+			app:theme="@style/ThemeOverlay.AppCompat.ActionBar">
+
+			<ImageView
+				android:layout_width="50dp"
+				android:layout_height="50dp"
+				android:src="@drawable/ic_app_logo"/>
+
+			<TextView
```

Comment:
There's no need for adding `TextView` and `ImageView` inside Toolbar, to display the title and app icon you may use this:
`getSupportActionBar().setDisplayShowHomeEnabled(true);`
Along with
`getSupportActionBar().setIcon(R.drawable.ic_launcher);`
`getSupportActionBar().setTitle("Title");`